webpack/assets/javascripts/react_app/components/common/forms/Checkbox.js

```diff
@@ -3,14 +3,22 @@ import PropTypes from 'prop-types';
 
 import { noop } from '../../../common/helpers';
 import CommonForm from './CommonForm';
+import { makeOnChangeHanler } from './helpers';
 
-const Checkbox = ({ className, checked, onChange, label, disabled }) => (
+const Checkbox = ({
+  className,
+  checked,
+  onValueChange,
```

Comment:
so just leave Checkbox in unsync state with the other components? I got no problem if we throw it in the future, but otherwise I would like to keep it synced :)